addons/account/models/account_journal_dashboard.py

```diff
@@ -713,23 +713,23 @@ def _get_draft_sales_purchases_query(self):
 
     def _get_open_sale_purchase_query(self, journal_type):
         assert journal_type in ('sale', 'purchase')
-        query = self.env['account.move.line']._where_calc([
-            ('move_id', 'in', self.env['account.move']._where_calc([
-                *self.env['account.move.line']._check_company_domain(self.env.companies),
-                ('journal_id', 'in', self.ids),
-                ('payment_state', 'in', ('not_paid', 'partial')),
-                ('move_type', '=', 'out_invoice' if journal_type == 'sale' else 'in_invoice'),
-                ('state', '=', 'posted'),
+        query = self.env['account.move']._where_calc([
+            *self.env['account.move.line']._check_company_domain(self.env.companies),
```

Comment:
I think it should be
```suggestion
            *self.env['account.move']._check_company_domain(self.env.companies),
```